client/js/src/client.ts

```diff
@@ -60,10 +61,11 @@ export class Client {
 		return fetch(input, init);
 	}
 
-	eventSource_factory(url: URL): EventSource | null {
+	eventSource_factory(url: URL): EventSource {
 		if (typeof window !== undefined && typeof EventSource !== "undefined") {
 			return new EventSource(url.toString());
 		}
+		// @ts-ignore
 		return null; // todo: polyfill eventsource for node envs
 	}
```

Comment:
This tying is temporary but was causing problems for some components. Ignored it until we polyfill.